docs/_documentations/troubleshooting.md

```diff
@@ -292,6 +292,18 @@ If you have the Codewind Language Server for Node.js Profiling extension enabled
 
 **Workaround:** Review the preceding list and ensure that your project conforms to all of the items in the list.
 
+<!--
+Action/Topic: Understanding application metrics
+Issue type: info
+Issue link: https://github.com/eclipse/codewind/issues/1251
+Info added in 0.10.0.
+-->
+## Setting Codewind server logging levels
+To assist with problem determination, raise the default Codewind server log level to **Debug** or **Trace** with the `cwctl loglevels` command.
+
+- In Eclipse, set your preference to enable support features. If you enable the preference, the menus include setting the Codewind server log level items.
```

Comment:
I still think we should tell the user to right click on the connection in the Codewind Explorer view so they know what menu to use or `the connection menu in the Codewind Explorer view`.  Just saying `the menus` is confusing.